pkg/core/localize/locales/en/cmd/serviceaccount_create.en.toml

```diff
@@ -31,6 +32,9 @@ $ rhoas service-account create --overwrite
 
 # Create a service account and save credentials to a custom file location
 $ rhoas service-account create --output-file=./service-acct-credentials.json
+
+# Create a service account and save the credentials in a credentials.properties file
```

Comment:
`credentials.properties` is generated for both `properties` and `java-kafka-properties`, it will be better to mention the utility.
```suggestion
# Create a service account and save the credentials in a properties file suitable for Java Kafka client
```